zstd/internal/xxhash/xxhash_unsafe.go

```diff
@@ -0,0 +1,57 @@
+// +build !appengine
```

Comment:
Remove the unsafe package stuff. It is not needed and I don't want to import unsafe.